ParckingApplication/Parcking.cs

```diff
@@ -1,7 +1,52 @@
+using System;
+using System.Net.Sockets;
+using System.Runtime.CompilerServices;
+using Microsoft.VisualBasic;
 
 namespace ParckingApplication;
 
 public class Parking
-{
+{   
+    public int MaxSlots = 4;
     
+
+    public List<ParkedCar> ListOfParkedCar = new List<ParkedCar> { new ParkedCar("Bogdan",  "abc" ), new ParkedCar("Andrei",  "abcde")};
+    
+
+    public void AddFunction(string userName,string? carNumber = null)
```

Comment:
Redenumeste functia in ceva sugestiv, fara sa folosesti cuvantul 'Function', gen AddToParking